avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/Avro2TF.scala

```diff
@@ -88,6 +88,8 @@ object Avro2TF {
    */
   case class SparseVector(indices: Seq[Long], values: Seq[Float])
 
+  case class RankedSparseVector(indices: Seq[Seq[Long]], values: Seq[Float])
```

Comment:
I find it confusing that `indices` is 2-dimensional while `values` is 1-dimensional. Why mix and match in this fashion?

Reply:
It's based on https://www.tensorflow.org/api_docs/python/tf/io/SparseFeature. The indices are column major ordered and values are always flattened. BTW Please check new pull request (#62) I will close this pull request